src/main/java/edu/cornell/kfs/module/cam/document/service/impl/CuAssetGlobalServiceImpl.java

```diff
@@ -19,5 +25,42 @@ protected Asset setupAsset(final AssetGlobal assetGlobal, final AssetGlobalDetai
         return asset;
 
     }
+    
+    @Override
+    protected void setupAssetLocationOffCampus(final AssetGlobalDetail assetGlobalDetail, final Asset asset) {
```

Comment:
Should there be a comment on this method that explains why it's being overridden, especially since there only appears to be one line that differs from the content in base code?

Reply:
ok